### PR TITLE
[Coverity] Fix unreachable codes

### PR DESCRIPTION
**Pull request by dongju-chae**
This patch fixes unreachable codes reported by coverity.

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c [hunk 1]
```diff
@@ -504,6 +504,7 @@ nnfw_setInputDim (const GstTensorFilterProperties * prop, void **private_data,
   /** Return -ENOENT till nnfw supports set input dimension internally */
   return -ENOENT;
 
+#if 0
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:507]: Do not add ```#if 0```. It deteriorates readability greatly. If these codes are really useless, delete them.

Comment (again4you) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:507]: I agreed with MJ's opinion.

Comment (myungjoo) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:507]: If the below codes are waiting for unimplemented feature, it'd better have some meaningful text along with the ```#if```. e.g., ```#ifndef FEATURE_XXX_IMPLEMENTED```.

Comment (dongju-chae) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:507]: Got it.

**Review by myungjoo — APPROVED** [review]
LGTM

File: ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c [hunk 2]
```diff
@@ -41,6 +41,7 @@
 #define NNFW_NEON_BACKEND "acl_neon"
 #define NNFW_SRCN_BACKEND  "srcn"
 #define NNFW_DEFAULT_BACKEND NNFW_CPU_BACKEND
+#define NNFW_FEATURE_SETINPUTDIM_IMPLEMENTED (0)
```
Comment (gichan-jang) [inline ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:44]: Why is the macro name NNFW_FEATURE_SETINPUTDIM_IMPLEMENTED?

Comment (dongju-chae) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:44]: Any suggestion?

Comment (gichan-jang) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:44]: It seems appropriate.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**